FAQ.md

```diff
@@ -0,0 +1,55 @@
+# FAQ
+
+**Table of Contents:**
+1. [Why does the size of the quantized model remain the same as the original model size?](#1-why-does-the-size-of-the-quantized-model-remain-the-same-as-the-original-model-size)
+2. [Why does loading a quantized exported model from a file fail?](#2-why-does-loading-a-quantized-exported-model-from-a-file-fail)
+3. [Why am I getting a torch.fx error?](#3-why-am-i-getting-a-torchfx-error)
+
+
+## 1. Why does the size of the quantized model remain the same as the original model size?
```

Comment:
Cosmetics:
Add space between the table of contents and the questions.
Minimize the subtitles. it's huge. try ### or ####